### PR TITLE
Release notes for 21.2.1

### LINKED ISSUE #10727: v21.2.1 Release Notes
Prep date: tbd
Release date: Nov 29, 2021
SHA candidate: tbd
Release Manager: tbd
Issue: tbd
[Docs To Do](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-16.Updatedocs):

- [ ] Run script to generate release notes
- [ ] Update docs site
- [ ] Edit release notes - Additions/changes/omissions to the release notes should be left in this issue's comments
- [ ] Once PR is merged / release notes are live, notify @meagan-g in the #releases Slack channel
- [ ] Email releases@ with link to published release notes
- [ ] Generate release notes issues (cc @jseldess)

### PR DESCRIPTION
**Pull request by kathancox**
Closes #10727 

Release notes for extraordinary release 21.2.1

### CODE REVIEW

**Review by rmloveland — APPROVED** [review]
LGTM

but it probably needs a re-push to get the tests green and the deploy ready (probably not your PR just some typical cloud flakes)

**Comment by kathancox** [comment]
> LGTM
> 
> but it probably needs a re-push to get the tests green and the deploy ready (probably not your PR just some typical cloud flakes)

I believe it's because I've linked to the Technical advisory that's not published yet. Thanks for the review Rich.

File: releases/v21.2.1.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- [Backups](../v21.2/take-full-and-incremental-backups.html) taken while a cluster contains a mix of v21.2 and v21.1 nodes may fail. [Upgrading](../v21.2/upgrade-cockroach-version.html) the entire cluster to v21.2 should resolve the issues. The [technical advisory 72839](../advisories/a72839.html) provides more information about possible [remediations](../advisories/a72839.html#mitigation). The error returned after a backup failure in this case now also directs the user to the technical advisory. [#72880][#72880]
+- Fixed a bug that caused a [full-cluster backup](../v21.2/backup.html#backup-a-cluster) to fail while upgrading from v21.1 to v21.2. This caused an error, because the `system.tenant_usage` table, which is present in v21.2, is not present in v21.1. [#72840][#72840]
+- Fixed a bug where cluster backups were backing up opt-out system tables unexpectedly. [#71368][#71368]
+
+### Contributors
+
+This release includes 0 merged PRs by 6 authors.
```
Comment (stbof) [inline releases/v21.2.1.md:48]: @kathancox Is `0` merged PRs correct?

Comment (kathancox) [reply releases/v21.2.1.md:48]: Ahh yes, missed this. Looks like this is as a result of commits being cherry-picked for this particular release. I will make a change after @rmloveland's review. Thanks!

**Comment by rmloveland** [comment]
still LGTM

**Pull request merged**